core/src/main/java/org/ethereum/beacon/core/types/Bitfield.java

```diff
@@ -0,0 +1,23 @@
+package org.ethereum.beacon.core.types;
+
+import org.ethereum.beacon.ssz.annotation.SSZSerializable;
+import tech.pegasys.artemis.util.uint.UInt64;
+
+@SSZSerializable(serializeAs = UInt64.class)
+public class Bitfield extends UInt64 {
+
+  public static final Bitfield ZERO = new Bitfield(UInt64.ZERO);
+
+  public Bitfield(UInt64 uint) {
```

Comment:
There are two bitfields in the domain. This one is `UInt64` based and used as a type for `justificationBitfield` field. Another one is based on `BytesValue` and has arbitrary length, used in `AttestationData`.

IMO, `Bitfield` name matches better for the latter case. As for `justificationBitfield` I really have no idea except `Bitfield64`.

Reply:
I've started to use `Bitfield` for participationBitfield, and it's definitely more sense to use it there

Reply:
Sounds reasonable. Done!